testbed/tests/perf_test.go

```diff
@@ -65,14 +68,51 @@ func TestIdleMode(t *testing.T) {
 	tc.Sleep(10 * time.Second)
 }
 
-func Test10kSPS(t *testing.T) {
-	tc := testbed.NewTestCase(t)
+// createConfigFile creates a collector config file that corresponds to the
+// exporter and receiver used in the test and returns the config file name.
+func createConfigFile(exporter testbed.TraceExporter, receiver testbed.Receiver) string {
+	config := fmt.Sprintf(`
+receivers:%v
+exporters:%v
+processors:
+  queued_retry:
+
+service:
+  pipelines:
+    traces:
+      receivers: [%v]
+      processors: [queued_retry]
+      exporters: [%v]
+`, exporter.GenConfigStr(), receiver.GenConfigStr(), exporter.ProtocolName(), receiver.ProtocolName())
```

Comment:
position of exporters and receiver swapped?

Reply:
No, it is actually correct. Added a comment to explain why. :-)

Reply:
thanks for adding the comment.